src/plugins/ctd/parser.py

```diff
@@ -236,6 +240,9 @@ def load_data(data_dir):
             my_geneset['_id'] = chemical_id + "-" + tax_id
```

Comment:
Suggest you change `-` in field values to `_`, since it means "OR" in elasticsearch syntax. (it can be escaped with `\`, but it's better to avoid it.)

All the reserved characters are: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /